mkosi/__init__.py

```diff
@@ -1751,7 +1802,9 @@ def build_uki(
         options += ["--ro-bind", initrd, workdir(initrd)]
 
     with complete_step(f"Generating unified kernel image for kernel version {kver}"):
-        run_ukify(context, stub, output, cmdline=cmdline, arguments=arguments, options=options)
+        return run_ukify(
+            context, stub, output, cmdline=cmdline, arguments=arguments, options=options, json_out=json_out
+        )
```

Comment:
Nit:
```suggestion
        return run_ukify(
            context, 
            stub, 
            output, 
            cmdline=cmdline, 
            arguments=arguments, 
            options=options, 
            json_out=json_out,
        )
```

Reply:
tried that first, but the linter rejects it and wants it this way

Reply:
Did you have the trailing command after `json_out`? Without it `ruff format` will make it a single line, with it, it comes out like in the suggestion.

Reply:
that was probably it